.nvmrc

```
@@ -1 +1 @@
-v16.18.0
\ No newline at end of file
+v20.13.1
```

How about use a loose range?

```suggestion
lts/iron
```

https://github.com/search?q=+lts%2FIron+path%3A.nvmrc&type=code